app/controllers/application_controller.rb

```diff
@@ -542,7 +542,7 @@ def user_duplicate_profiles_detected?
     return false unless sp_eligible_for_one_account?
     profile = current_user&.active_profile
     DuplicateProfileConfirmation.where(
-      profile_id: profile.id,
+      profile_id: profile&.id,
```

Comment:
This column can't be `null`, so could probably `return false` if `profile` is `nil` above this.
```suggestion
      profile_id: profile&.id,
```